### PR TITLE
sci-libs/hipBLAS: ROCm BLAS marshalling library

### PR DESCRIPTION
**Pull request by littlewu2508**
hipBLAS is a front end of rocBLAS, which is the dependency of some ROCm
GPGPU package like cupy.

Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: Yiyang Wu <xgreenlandforwyy@gmail.com>

### CODE REVIEW

**Comment by littlewu2508** [comment]
@justxi I'd like to land hipBLAS to the portage tree, so I pulled your ebuild in https://github.com/justxi/rocm and bump its version to 4.3.0 with minor fixes as usual. Is it OK to add you as the package maintainer?

**Comment by littlewu2508** [comment]
Oops seems that rocSOLVER is not merged, so just hang this PR and wait for rocSOLVER

File: sci-libs/hipBLAS/hipBLAS-4.3.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+DEPEND="${RDPEND}"
+BDEPEND=""
+
+S="${WORKDIR}/hipBLAS-rocm-${PV}"
+
+src_prepare() {
+	sed -e "s:<INSTALL_INTERFACE\:include:<INSTALL_INTERFACE\:include/hipblas/:" -i library/src/CMakeLists.txt || die
+	sed -e "/PREFIX hipblas/d" -i library/src/CMakeLists.txt || die
+	sed -e "/rocm_install_symlink_subdir( hipblas )/d" -i library/src/CMakeLists.txt || die
+	sed -e "s:get_target_property( HIPHCC_LOCATION hip\:\:hip_hcc IMPORTED_LOCATION_RELEASE ):get_target_property( HIPHCC_LOCATION hip\:\:hip_hcc IMPORTED_LOCATION_GENTOO ):" -i library/src/CMakeLists.txt
```
Comment (heroxbd) [inline sci-libs/hipBLAS/hipBLAS-4.3.0.ebuild:27]: This line is too long.  Try shorten it.

Can it be
```
/HIPCC_LOCATION/s/IMPORTED_LOCATION_RELEASE/IMPORTED_LOCATION_GENTOO/
```
?

Comment (littlewu2508) [reply sci-libs/hipBLAS/hipBLAS-4.3.0.ebuild]: I removed this line because it's does not modified anything.

**Comment by heroxbd** [comment]
CI:

```
  | PkgMetadataXmlEmptyElement | metadata.xml: empty element 'use' on line 15
```

**Comment by littlewu2508** [comment]
> CI:
> 
> ```
>   | PkgMetadataXmlEmptyElement | metadata.xml: empty element 'use' on line 15
> ```

Fixed

**Pull request closed without merging**